tests/models/test_dagrun.py

```diff
@@ -17,161 +17,160 @@
 # under the License.
 
 import datetime
-import unittest
+from typing import Mapping, Optional
 from unittest import mock
 from unittest.mock import call
 
+import pendulum
 import pytest
-from parameterized import parameterized
+from sqlalchemy.orm.session import Session
 
-from airflow import models, settings
-from airflow.models import DAG, DagBag, DagModel, TaskInstance as TI, clear_task_instances
-from airflow.models.dagrun import DagRun
+from airflow import settings
+from airflow.models import DAG, DagBag, DagModel, DagRun, TaskInstance as TI, clear_task_instances
 from airflow.operators.dummy import DummyOperator
 from airflow.operators.python import ShortCircuitOperator
 from airflow.serialization.serialized_objects import SerializedDAG
 from airflow.stats import Stats
 from airflow.utils import timezone
 from airflow.utils.callback_requests import DagCallbackRequest
 from airflow.utils.dates import days_ago
-from airflow.utils.state import State
+from airflow.utils.state import DagRunState, State, TaskInstanceState
 from airflow.utils.trigger_rule import TriggerRule
 from airflow.utils.types import DagRunType
-from tests.models import DEFAULT_DATE
+from tests.models import DEFAULT_DATE as _DEFAULT_DATE
 from tests.test_utils.db import clear_db_dags, clear_db_pools, clear_db_runs
 from tests.test_utils.mock_operators import MockOperator
 
+DEFAULT_DATE = pendulum.instance(_DEFAULT_DATE)
 
-class TestDagRun(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.dagbag = DagBag(include_examples=True)
 
-    def setUp(self):
+class TestDagRun:
+    dagbag = DagBag(include_examples=True)
```

Comment:
Is this a problem that it would be loaded at collection/file parse time and ot only when this test is actually being run?

Reply:
I wondered about that but there’s no functional difference.